src/gdcdictionary/schemas/sample.yaml

```diff
@@ -195,6 +195,7 @@ properties:
       - Hard Palate
       - Head & Neck
       - Head - Face Or Neck, Nos
```

Comment:
And this one stays here?

Reply:
Correct, it stays but an entry is created under 'deprecated_enum' to indicate it has been deprecated from use.